mkdocs.yml

```diff
@@ -193,6 +193,9 @@ pages:
         - Generic Logs: integration/generic-logs-integration.md
         - Apache Logs: integration/apache-logs.md
         - AWS ECS: integration/ecs-logs.md
```

Comment:
maybe you want aws- prefix, but if PITA to rename now just ignore me.

Reply:
done